### PR TITLE
feat(maven-private-repo) add doc to configure private maven repo

### PR DESCRIPTION
**Pull request by alexisries**

### CODE REVIEW

**Review by rbioteau — COMMENTED** [review]
Maybe add a note to explain that if more than one repository is needed, they should contact the support ?

File: modules/ROOT/pages/Continuous_Delivery_Configuring_your_Continuous_Delivery_Platform.adoc [hunk 1]
```diff
@@ -19,6 +19,17 @@ image:images/CredentialsJobResult.png[JobResult]
 
 CAUTION: If your Git repository is hosted on GitHub.com, beginning August 13, 2021 password-based authentication will no longer be accepted. Instead, token-based authentication will be required using a https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token[personal access token]. Therefore you will have to enter your GitHub username in the **bonitaGitUsername** field and your authentication token in the **bonitaGitPassword** field of the "Configure my credentials" job.
 
+== Maven repository credentials
+
+This job allows you to configure you access to a private Maven repository.
```
Comment (JeremJR) [inline modules/ROOT/pages/Continuous_Delivery_Configuring_your_Continuous_Delivery_Platform.adoc:24]:
```suggestion
This job allows you to configure your access to a private Maven repository.
```

File: modules/ROOT/pages/Continuous_Delivery_Configuring_your_Continuous_Delivery_Platform.adoc [hunk 2]
```diff
@@ -31,6 +31,8 @@ This job allows you to configure you access to a private Maven repository.
 . Make sure the job was successful:
 image:images/ConfigureMavenJobResult.png[JobResult]
 
+NOTE: The job only allows setting up a single private Maven repository, to set up multiple repos please create a new case on the Customer Service Center.
```
Comment (rbioteau) [inline modules/ROOT/pages/Continuous_Delivery_Configuring_your_Continuous_Delivery_Platform.adoc:34]:
```suggestion
NOTE: The job only allows setting up a single private Maven repository, to set up multiple repositories please create a new case on the Customer Service Center.
```

File: modules/ROOT/pages/Continuous_Delivery_Configuring_your_Continuous_Delivery_Platform.adoc [hunk 3]
```diff
@@ -21,7 +21,7 @@ CAUTION: If your Git repository is hosted on GitHub.com, beginning August 13, 20
 
 == Maven repository credentials
 
-This job allows you to configure you access to a private Maven repository.
+This job allows you to configure your access to a private Maven repository.
 
 . Click on the image:images/JenkinsPlayButton.png[CDPlayButton] of the "Configure my maven private repo" job.
```
Comment (rbioteau) [inline modules/ROOT/pages/Continuous_Delivery_Configuring_your_Continuous_Delivery_Platform.adoc:26]: Should the jenkins job not use an abbreviation for repository ?

Comment (alexisries) [reply modules/ROOT/pages/Continuous_Delivery_Configuring_your_Continuous_Delivery_Platform.adoc]: Yes, I changed the name of the job on Jenkins, thank you

**Review by JeremJR — APPROVED** [review]

**Pull request merged**